config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
there is no such object as a `validation` to create. This service is analogous to `search`: it's something you do, not something you act upon. I.e. it's not REST-style, it's RPC-style.